### PR TITLE
This PR solves issue #28 refactor code and update readme

### PR DESCRIPTION
**Pull request by mansikulkarni96**

### CODE REVIEW

File: app/__init__.py [hunk 1]
```diff
@@ -37,29 +36,44 @@
     }
 )
 
-# Route for creating workspaces
+
 @app.route('/workspace/create', methods=['POST'])
-def linchpin_init():
+def linchpin_init() -> Response:
```
Comment (samvarankashyap) [inline app/__init__.py:41]: @mansikulkarni96 is this correct syntax ??

Comment (mansikulkarni96) [reply app/__init__.py:41]: As of python 3.5 it is the syntax to mention method return type:
https://stackoverflow.com/questions/5336320/how-to-know-function-return-type-and-argument-types/31152839
I saw some examples in which REST apis return Response in return type.

Comment (samvarankashyap) [reply app/__init__.py:41]: Surprising I have not used this syntax yet . 
I better note it down :)

**Review by samvarankashyap — APPROVED** [review]
LGTM

**Pull request merged**